Specify number of inverters in config.ini (#148)
henne49/dbus-opendtu: README.md

|-------------------- | ------------- |
| SignOfLifeLog  | Time in minutes how often a status is added to the log-file `current.log` with log-level INFO |
| NumberOfTemplates | Number ob Template Inverter to query |

> | DTU | Which DTU to be used ahoy, opendtu or template REST devices Valid options: opendtu, ahoy, template |

Reviewer: Nice :+1:

> | QueryOnlyFirstOfDTUInverter | Amount of Inverters you to query. Set a value larger than "0" when not all inverters should be considered. Please assure that the order is correct in the DTU, we can only extract the first one in a row. |

Reviewer: ```suggestion
| QueryOnlyFirstOfDTUInverter | Number of inverters you want to query. if you want to query all inverters, set this to 0. Please make sure that the order in the DTU is correct, we can only extract the first one in a row. |
```
The text is quite long, maybe it can be shortened and the rest could be in a footnote.  See  [Footnotes](https://github-blog.translate.goog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/?_x_tr_sl=en&_x_tr_tl=de&_x_tr_hl=de&_x_tr_pto=sc)
